i18n/userguide/fr.po

```diff
@@ -770,40 +770,38 @@ msgstr ""
 
 #. NOTE: markdown formatting
 #: ../../frescobaldi_app/userguide/documentfonts_command.md:44
-#, fuzzy
 msgid ""
 "*Load notation-fonts package*: The same consideration regarding the "
 "*notation-fonts* package."
 msgstr ""
-"*Charger le paquetage notation-fonts* : Même considération pour le paquet "
+"*Charger le paquet `notation-fonts`* : Même considération pour le paquet "
 "*notation-fonts*."
 
 #. NOTE: markdown formatting
 #: ../../frescobaldi_app/userguide/documentfonts_command.md:46
-#, fuzzy
 msgid ""
 "*Load font extensions:* Some fonts include additional glyphs beyond the "
 "regular LilyPond coverage. openLilylib provides stylesheets with supporting "
 "functions making the additional features available as commands or e.g. "
 "articulations. If this option is checked font extensions are loaded if "
 "available (otherwise nothing will happen)."
 msgstr ""
-"*Charger les extensions de police :* Certaines polices comprennent des "
-"glyphes supplémentaires au-delà de la couverture normale de LilyPond. "
-"openLilylib fournit des feuilles de style avec des fonctions de support "
-"rendant les caractéristiques supplémentaires disponibles comme commandes ou, "
-"par exemple, articulations. Si cette option est cochée, les extensions de "
-"police sont chargées si elles sont disponibles (sinon rien ne se passe)."
+"*Charger les extensions de police de caractère :* Certaines polices "
+"comprennent des glyphes supplémentaires au-delà de la couverture normale de "
+"LilyPond. openLilylib fournit des feuilles de style avec des fonctions de "
+"support rendant les caractéristiques supplémentaires disponibles comme "
+"commandes ou, par exemple, articulations. Si cette option est cochée, les "
+"extensions de police sont chargées si elles sont disponibles (sinon rien ne "
+"se passe)."
 
 #. NOTE: markdown formatting
 #: ../../frescobaldi_app/userguide/documentfonts_command.md:51
-#, fuzzy
 msgid ""
 "*Font stylesheets:* openLilyLib provides default stylesheets for each "
 "supported font, adjusting the visual appearance (e.g. line thickness of "
 "various score items) to match the characteristics of the given font. By "
-"default these are loaded automatically by the font command "
-"*\\useNotationFont*. However, in some cases it may be necessary to *not* use "
+"default these are loaded automatically by the font command *"
```

Comment:
The changes in msgid strings of .po files don't make any sense, as they will get overwritten when the strings are extracted the next time. Translators should only work on msgstr strings: in fact PO editors usually prevent the editing of msgid strings. Anyway, I would just explain it to Olivier for next contributions; I'm not asking you to fix it in this PR.

Reply:
Or maybe Github is showing me the diff of the entire PR even if I entered the first commit? If the answer is yes, ignore my previous comment.